.circleci/config.yml

```diff
@@ -331,3 +379,19 @@ workflows:
       - pre-commit
       - python_lint
       - install_and_test_ubuntu
+  testpypi_nightly:
```

Comment:
Can you detail what this would look like for release to the real pypi? What would trigger the upload when we do a release ?

Reply:
It is not added yet, but it should be pretty easy to implement. I'll add it soon and ping you for the review.